funcx_endpoint/funcx_endpoint/endpoint/endpoint.py

```diff
@@ -90,8 +90,13 @@ def start_endpoint(
     endpoint_uuid : str
     """
     endpoint_dir = os.path.join(manager.funcx_dir, name)
-    endpoint_config = SourceFileLoader('config',
-                                       os.path.join(endpoint_dir, manager.funcx_config_file_name)).load_module()
+    try:
+        endpoint_config = SourceFileLoader('config',
+                                           os.path.join(endpoint_dir, manager.funcx_config_file_name)).load_module()
+    except Exception:
+        manager.logger.exception('Need to update the out of date config. '
```

Comment:
Can we change to "Your config is out of date and needs updating. Please refer to the docs at ..."

Reply:
I would give this a bit more leeway. "funcX v0.2.0 made several changes non-backwards compatible changes to the config. Your config might be out of date. Please refer to the docs at ...". @ryanchard what do you think?